.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+package-lock.json
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly remove the package-lock.json file from the gitignore as this needs to be visible in your repository.